ui/src/pages/QuickSetup/RequestServicesPage.tsx

```diff
@@ -30,22 +34,44 @@ import { RequestsProvider } from '../../context/RequestsContext';
 
 import { retrieveUserParties } from '../../Parties';
 import { IconCheck, InformationIcon } from '../../icons/icons';
+import { Account } from '@daml.js/da-marketplace/lib/DA/Finance/Types';
 
 import QuickSetupPage from './QuickSetupPage';
 import { MenuItems, LoadingWheel } from './QuickSetup';
+import { NewAccount } from '../custody/New';
+import { CreateEvent } from '@daml/ledger';
+import { AssetSettlementRule } from '@daml.js/da-marketplace/lib/DA/Finance/Asset/Settlement';
+import { AllocationAccountRule } from '@daml.js/da-marketplace/lib/Marketplace/Rule/AllocationAccount';
+import AccountSelectionModal, { AccountType, AccountInfos } from './AccountSelectionModal';
+
+type AccountsForServices = {
+  clearingAccount?: Account;
+  marginAccount?: Account;
+  tradingAccount?: Account;
+  tradingAllocAccount?: Account;
+  biddingAccount?: Account;
+  biddingAllocAccount?: Account;
+  auctionAccount?: Account;
+  auctionAllocAccount?: Account;
+  receivableAccount?: Account;
+};
 
 export interface IRequestServiceInfo {
   provider?: string;
   customer?: string;
   services?: ServiceKind[];
+  accounts?: AccountsForServices;
 }
 
-// without the ability to create accounts from quick setup, these requests are the only ones supported at this stage
 const SUPPORTED_REQUESTS = [
   ServiceKind.MARKET_CLEARING,
   ServiceKind.LISTING,
   ServiceKind.CUSTODY,
   ServiceKind.ISSUANCE,
+  ServiceKind.CLEARING,
+  ServiceKind.TRADING,
+  ServiceKind.AUCTION,
+  ServiceKind.BIDDING,
 ];
```

Comment:
currently there's still `Settlement`, `Distributor` and `Regulator` in `ServiceKind` in addition to those...Settlement and Distributor are now in `RoleKind` so we can probably get rid of those anyway, `Regulator` I'm not sure about though, it's done elsewhere manually but it may be useful to have in the `ServicesContext` anyway